experimental/packages/otlp-proto-exporter-base/src/platform/browser/OTLPProtoExporterBrowserBase.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { diag } from '@opentelemetry/api';
+import { ServiceClientType } from '../types';
+import {
+  OTLPExporterBrowserBase as OTLPExporterBaseMain,
+  // CompressionAlgorithm,
+  OTLPExporterError,
+  OTLPExporterConfigBase
+} from '@opentelemetry/otlp-exporter-base';
+import { send } from './util';
```

Comment:
I wonder what was the reason `send` function was moved into a separate util, and not OTLPProtoExporterBrowserBase itself. Are there any benefits to doing it?

Reply:
See https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/otlp-grpc-exporter-base/src/OTLPGRPCExporterNodeBase.ts#L78-L79 for the reason for Node.js grpc instrumentation.

I think it is fine to move the util function as a member method for the web, as the `send` function is accessing many members of the `collector` and not depends on external modules.